tests/test_lambda.py

```diff
@@ -193,6 +194,45 @@ def test_lambda_config_source(self):
         )
         self.assertTrue("c7n:Policy" in resources[0])
 
+    def test_post_finding(self):
+        factory = self.replay_flight_data('test_lambda_post_finding')
+        p = self.load_policy({
+            'name': 'lambda',
+            'resource': 'aws.lambda',
+            'actions': [
+                {'type': 'post-finding',
+                 'types': [
+                     'Software and Configuration Checks/OrgStandard/abc-123']}]},
+            session_factory=factory, config={'region': 'us-west-2'})
+        functions = p.resource_manager.get_resources([
+            'custodian-ec2-ssm-query'])
+        rfinding = p.resource_manager.actions[0].format_resource(functions[0])
+        self.maxDiff = None
+        code = rfinding['Details']['AwsLambdaFunction'].pop('Code', None)
+        self.assertEqual(
+            rfinding,
+            {'Details': {'AwsLambdaFunction': {
+                'CodeSha256': 'Pq32lM46RbVovW/Abh14XfrFHIeUM/cAEC51fwkf+tk=',
+                'Handler': 'custodian_policy.run',
+                'LastModified': '2019-07-29T22:37:20.844+0000',
+                'RevisionId': '8bbaf510-0ae1-40a5-8980-084bebd3f9c6',
+                'Role': 'arn:aws:iam::644160558196:role/CloudCustodianRole',
+                'Runtime': 'python3.7',
+                'Timeout': 900,
+                'TracingConfig': {'Mode': 'PassThrough'},
+                'Version': '$LATEST',
+                'VpcConfig': {'SecurityGroupIds': [],
+                              'SubnetIds': []}}},
+             'Id': 'arn:aws:lambda:us-west-2:644160558196:function:custodian-ec2-ssm-query',
+             'Partition': 'aws',
+             'Region': 'us-west-2',
+             'Tags': {'custodian-info': 'mode=config-rule:version=0.8.44.2'},
+             'Type': 'AwsLambdaFunction'})
+        rfinding['Details']['AwsLambdaFunction']['Code'] = code
```

Comment:
not sure about this line here because the ['Details']['AwsLambdaFunction']['Code'] was removed in 211. What was the reason for popping it off above?

Reply:
because its a very, very long url (1599 chars) and doesn't add meaningful value to check the literal content, but given the other discussion on code, I'll be looking at redoing the code section anyways.